src/app/components/breadcrumbs/service/terra-breadcrumbs.service.ts

```diff
@@ -298,27 +247,57 @@ export class TerraBreadcrumbsService
         // search breadcrumb
         let index:number = breadcrumbList.indexOf(breadcrumb);
 
-        breadcrumbList.splice(index, 1);
+        if(breadcrumbContainer.currentSelectedBreadcrumb === breadcrumb)
+        {
+            // search for previous breadcrumb
+            let firstBreadcrumb:TerraBreadcrumb = breadcrumbList[0];
```

Comment:
choose breadcrumbList[1] instead, when the current selected breadcrumb and the one being deleted is breadcrumbList[0]